src/main/resources/application.yaml

```diff
@@ -23,6 +22,10 @@ spring:
     exclude: org.springframework.boot.autoconfigure.mustache.MustacheAutoConfiguration
   jackson:
     default-property-inclusion: NON_EMPTY
+  cloud:
+    openfeign:
+      okhttp:
+        enabled: true
```

Comment:
There is no feign clients

Reply:
Then you should have this property with false value in the module config. If it will not help, then changes in folio-spring-base is needed.